ansible/roles/test/files/ptftests/arista.py

```diff
@@ -153,15 +154,19 @@ def run(self):
                 }
 
         attempts = 60
+        log_present = False
         for _ in range(attempts):
             log_output = self.do_cmd("show log | begin %s" % log_first_line)
             log_lines = log_output.split("\r\n")[1:-1]
             log_data = self.parse_logs(log_lines)
-            if len(log_data) != 0:
+            if (self.reboot_type == 'fast-reboot' and\
```

Comment:
>\ [](start = 53, length = 1)

Add one more blank before '\', or aligned them in multiple lines

Reply:
added space